guides/source/active_storage_overview.md

```diff
@@ -462,6 +462,19 @@ end
 <%= image_tag user.video.preview(:thumb) %>
 ```
 
+If your variants take too long to generate on the fly, you
+can specify that rails should generate them ahead of time:
```

Comment:
The reasoning is not so much for the "too long" as it may be required before the job runs anyway. It's more when you are certain you'll need it, see: https://github.com/rails/rails/pull/47473

But otherwise, yes it's valuable to document that I think

Reply:
Thanks very much for pointing that out to me -- I have updated the text to state the original reason for the feature.